Snowdog_Components/components/Organisms/footer/_footer.scss

```diff
@@ -11,7 +11,15 @@
             padding: $footer__bottom-bar-handler-padding--checkout;
 
             @include mq($screen-m) {
-                padding: 0;
+                padding: $footer__bottom-bar-handler-padding--checkout\@medium;
+            }
+
+            @include mq($screen-l) {
+                padding: $footer__bottom-bar-handler-padding--checkout\@large;
+            }
+
+            @include mq($screen-xl) {
+                padding: $footer__bottom-bar-handler-padding--checkout\@xl;
```

Comment:
Check discussion on chat as we have different naming for "extra large"
xl, xlarge, extra-large. Personally I think we should use `\@extra-large` as we don't use "\@l" for (screen-l).

Reply:
fixed